ufo/user_test.py

```diff
@@ -246,40 +240,42 @@ def testAddUsersPostManualHandler(self, mock_add):
     response = self.client.post(flask.url_for('add_user'), data=data,
                                 follow_redirects=False)
 
+    matched_count = models.User.query.filter_by(email=FAKE_EMAILS_AND_NAMES[0]['email']).count()
```

Comment:
Ah, sorry, bit of clarification on the above line:

It tests that precisely one record now exists in the database that has that email address.

I'd personally argue that this is an improvement over the previous way of testing that it got added which was to make sure that `models.User.Add` got called at least once with no arguments.